src/Microsoft.Extensions.Logging.Abstractions/ILogger.cs

```diff
@@ -8,24 +8,24 @@ namespace Microsoft.Extensions.Logging
     /// <summary>
     /// Represents a type used to perform logging.
     /// </summary>
+    /// <remarks>Aggregates most logging patterns to a single method.</remarks>
     public interface ILogger
     {
         /// <summary>
-        /// Aggregates most logging patterns to a single method.
+        /// Writes an event to the logger.
```

Comment:
Not sure if `event` is the right word to use here. Can it just be `Writes a log entry`? @Eilon